front/src/components/competition/CompetitionCreationForm.tsx

```diff
@@ -221,10 +221,10 @@ export const CompetitionCreationForm = ({reset, minting, setMinting, setTokenId,
             <div className="form-ligne">
                 <label> Type de compétition :
                     <select name="type" onChange={e => setTypeCompetition(parseInt(e.target.value))}>
```

Comment:
Plutôt faire une fonction et écrire:
```suggestion
                    <select name="type" onChange={updateTypeCompetition}>
```

Avec:

```typescript
const updateTypeCompetition = (e) => setTypeCompetition(parseInt(e.target.value))
```

Dans la section "traitement" du composant (plus haut donc)
Ca va passer automatiquement l'évenement en premier argument. L'idée étant de mettre le moins de traitement dans la partie vue.